lib/win.js

```diff
@@ -92,13 +115,16 @@ function _copyAssetsToTmpFolder( options ) {
 
 /**
  * Replace / with \ to write file
+ * also, ensure drive character is upper case when run on windows platforms
  * paths for windows
  *
  * @param  {String} path path
  * @return {String}      windows path
  */
-function _windowsify( path ) {
-  return path.replace( /\//g, '\\' );
+function _windowsify( p ) {
+  p = path.win32.normalize(p);
```

Comment:
What version of node are you using? It relies on v0.12:
https://nodejs.org/api/path.html